### PR TITLE
Fix/calculate aircrew time

### PR DESCRIPTION
**Pull request by petermisik**
Updated HelperFunctions with the math to sum the row.

### CODE REVIEW

File: Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift [hunk 1]
```diff
@@ -32,8 +32,9 @@ class FlightTimeTableViewCell: UITableViewCell {
         instructor.text = crewMember.instructor
         evaluator.text = crewMember.evaluator
         other.text = crewMember.other
-        time.text = crewMember.time
```
Comment (Pete-Hoch) [inline Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: It looks like we never use this anymore. (But we do set it in the constructor.)

Should we remove the field from CrewMember?

File: Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift [hunk 2]
```diff
@@ -32,8 +32,9 @@ class FlightTimeTableViewCell: UITableViewCell {
...
+        time.text = Helper.airCrewTotalTimeCalculation(crewMember: crewMember)
```
Comment (Pete-Hoch) [inline Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: How about making this a function in CrewMember? Something like:

crewMember.totalAirTime().

**Review by Pete-Hoch — COMMENTED** [review]
Just a couple of questions. (And suggestions.)

File: Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift (hunk 2 above)
Comment (jbethanco) [reply Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: I'm having a hard time getting past our data model using strings for what should be doubles, ints, or dates. Have we thought about refactoring this? Converting from and to strings to do math isn't ideal.

Comment (Pete-Hoch) [reply Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: > I'm having a hard time getting past our data model using strings for what should be doubles, ints, or dates. Have we thought about refactoring this? Converting from and to strings to do math isn't ideal.

Understood, but we are likely to change persistence over to Core Data (or something) after 0.1.0 and we will be able to do that all at once. We can even add formatters to the text fields so the user can't even type a character in a numeric field. But again, this has all been very quickly put together and there is little reason to put a lot of time into the current UI when we plan to totally replace it by mid January.

At that point, we can address the data model and what makes the most sense.

Comment (jbethanco) [reply Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: Sounds good!

File: Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift (hunk 1 above)
Comment (petermisik) [reply Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:35]: I don't think we need to remove it from CrewMember.  This data needs to be attached to the model and part of the storage.

File: Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift (hunk 2 above)
Comment (petermisik) [reply Logging/Controllers/View Controllers/Split Time/FlightTimeTableViewCell.swift:47]: Changes are in the new PR.

**Review by jbethanco — CHANGES_REQUESTED** [review]
Looks good other than the tests needing an update. 
And just FYI: I believe Pete H is wanting two reviews prior to merges till 0.1.0

**Review by Pete-Hoch — CHANGES_REQUESTED** [review]
Need to fix up unit tests.

**Review by jbethanco — APPROVED** [review]

**Review by Pete-Hoch — APPROVED** [review]
Looks good!

**Pull request merged**